ably/transport/defaults.py

```diff
@@ -19,6 +19,7 @@ class Defaults:
     suspended_timeout = 60000
     comet_recv_timeout = 90000
     comet_send_timeout = 10000
+    realtime_request_timeout = 10
```

Comment:
is it possible for this to be represented in milliseconds like the other timeouts?

Reply:
The asyncio `wait_for` method expects the time in seconds

Reply:
I think it looks a bit weird if we have some timeouts which take milliseconds and others which take seconds, so I would prefer if we just divide it by `1000` before we pass it to `wait_for`